crates/sqlexec/src/remote/client.rs

```diff
@@ -173,15 +157,18 @@ impl RemoteClient {
     }
 
     /// Connect to a proxy destination.
-    pub async fn connect_with_proxy_destination(dst: ProxyDestination) -> Result<Self> {
-        Self::connect_with_proxy_auth_params(dst.dst.to_string(), dst.params, dst.tls_conf).await
+    pub async fn connect_with_proxy_destination(
+        dst: ProxyDestination,
+        ca_info: Option<CAInfo>,
```

Comment:
just thinking about all these API changes, but does it make sense to put the CAInfo in the ProxyDestination struct?